docs/source/interns/prepare-application.md

```diff
@@ -0,0 +1,128 @@
+(application-prep)=
+
+# Preparing your Final Application[^1]
+
+[^1]:
+    This section is heavily inspired by application guides from
+    [Zulip](https://zulip.readthedocs.io/en/latest/outreach/apply.html) and
+    [InterMine](http://intermine.org/internships/guidance/students-applying-outreachy/).
+    Thank you for developing such awesome resources!
+
+This section contains information on how to prepare your final application
+_specifically for the JupyterHub community_.
+
+```{seealso}
+Outreachy provides their own, more general
+[guide on how to complete your Final Application](https://www.outreachy.org/docs/applicant/#final-application).
+Please make to read through this as well.
+```
+
+Your first priority during the contribution period should be figuring out how
+to become an effective contributor. Start developing your final application only
+once you have experience iterating on your pull requests to get them ready to
+merge. This way, you'll have a much better idea of how much you can accomplish
+and how you may wish to tackle your chosen project.
+
+The JupyterHub team has a fluid approach to planning, which means it is likely
+that the set of issues described in the project description and your final
+application will change over the course of the internship. That's ok! These
+documents are not a strict commitment, either from you or your mentor.
+
+(application-prep:put-together)=
+
+## Putting together your application
+
+As you compile all of the information about you to put into your application,
+you should make sure that you cover the following topics:
+
+- Why you are applying:
+  - Why you're excited about working on JupyterHub
+  - What you're hoping to get out of your participation in JupyterHub
+  - What you're hoping to get out of working on the specific project you've chosen
+- Relevant experience:
+  - Summary of your **prior experience with the technologies** used by JupyterHub,
+    for example, Python
+  - Your **prior contributions to open source projects** (including pull requests,
+    bug reports, and so on) with links, if applicable (don't worry if this is
+    your first time contributing!)
+  - Any other **materials which will help us evaluate how you work**, such as
+    links to personal or school projects, along with brief descriptions
+- Your **contributions to JupyterHub**, including pull requests, bug reports, and
+  helping others in the community (with links to all materials)
+- Your answers to the [](application-prep:put-together:comm-qs)
+- A [](application-prep:put-together:proj-outline)
+
+(application-prep:put-together:comm-qs)=
+
+### Community-specific questions
+
+Please answer these questions in the relevant section of your application:
+
+1. **In your own words, what problem is the project attempting to solve?**
+   (_50 words max_)
+2. **In your own words, what is the main goal of the project?** (_50 words max_)
```

Comment:
Are these the best questions we could ask? What could we ask instead?

Reply:
I like the questions. Can you also consider one related to what success looks like for the intern within the specific project? I believe such a question would help in evaluating the applicant's understanding of the project's problem and deliverable(s).

Reply:
I agree that a question that gets at trying to describe concretely what the end of a successful internship would be helpful.

Reply:
I have added a new question to this section, what do you both think?

Reply:
The added question looks great :)